<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>IMU 제어기연결</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>IMU : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_imuLog">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>현재시간 : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="serverIP">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>192.168.101.15</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PORT">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>9003</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>361</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_object3d">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>441</width>
     <height>371</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
